<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1606</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Device">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>751</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <widget class="QLabel" name="red_ind">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>9</width>
       <height>17</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #913411;
border: 1px solid #adacac;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="blue_ind">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>9</width>
       <height>17</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #3859ab;
border: 1px solid #adacac;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="green_ind">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>20</y>
       <width>9</width>
       <height>17</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #17700f;
border: 1px solid #adacac;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="battery_ind">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>490</y>
       <width>26</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #f4f7f0;
background-color: #85cf1d;
color: #f4f7f0;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="Menu_select">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="minimumContentsLength">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>Select Menu Mode...</string>
     </property>
     <item>
      <property name="text">
       <string>Treatment Mode</string>
      </property>
      <property name="icon">
       <iconset theme="utilities-system-monitor"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Session History</string>
      </property>
      <property name="icon">
       <iconset theme="accessories-text-editor"/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Set Date/Time </string>
      </property>
      <property name="icon">
       <iconset theme="date"/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="power_btn">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>20</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px;
color: red;</string>
     </property>
     <property name="text">
      <string>DEVICE OFF</string>
     </property>
     <property name="icon">
      <iconset theme="application-exit"/>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pause_btn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>400</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PAUSE</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-pause"/>
     </property>
    </widget>
    <widget class="QPushButton" name="play_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>400</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PLAY</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start"/>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_btn">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>400</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-stop"/>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>371</width>
       <height>221</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="treatment_page">
      <widget class="QLabel" name="ConnLostTime_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>0</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Wait time:</string>
       </property>
      </widget>
      <widget class="QLabel" name="ConnLostTime_value">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>0</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>341</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <widget class="QLCDNumber" name="remainedTimeLCD">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>20</y>
             <width>201</width>
             <height>51</height>
            </rect>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
          </widget>
          <widget class="QProgressBar" name="session_progress">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>90</y>
             <width>321</width>
             <height>23</height>
            </rect>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
          <widget class="QTextBrowser" name="treatment_screen">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>130</y>
             <width>321</width>
             <height>51</height>
            </rect>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans'; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="session_hist_page">
      <widget class="QScrollArea" name="SessionHist_screen">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>331</width>
         <height>171</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>329</width>
          <height>169</height>
         </rect>
        </property>
        <widget class="QTableView" name="deviceSessionData">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>10</y>
           <width>321</width>
           <height>151</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="dateTime_page">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>331</width>
         <height>141</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>25</pointsize>
        </font>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2024</year>
         <month>12</month>
         <day>4</day>
        </datetime>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="Battery_label">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>500</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dateTimeDisplay">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="PC">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>40</y>
      <width>751</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>PC</string>
    </property>
    <widget class="QPushButton" name="uploadButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>400</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>UPLOAD SESSION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previousButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>PREVIOUS SESSION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>400</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>NEXT SESSION</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="sessionDisplay"/>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>521</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QRadioButton" name="chargeBtn">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>570</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
padding: 3px;
border-radius: 50px;</string>
    </property>
    <property name="text">
     <string>Charge</string>
    </property>
    <property name="icon">
     <iconset theme="network-wired"/>
    </property>
   </widget>
   <widget class="QPushButton" name="eeg_disconnect_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>570</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>EEG #1 DISCONNECTED</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eeg_reconnect_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>570</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>EEG #1 RECONNECTED</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1606</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
